lib/v2/neu/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.bmie.neu.edu.cn': {
+        _name: '东北大学医学与生物信息工程学院',
```

Comment:
```suggestion
        _name: '东北大学',
```
`_name` should match the H2 header of the docs